console/console-init/ui/src/modules/iot-device-detail/components/ConnectionGateway/ConnectionGateway.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import React from "react";
+import { Link } from "react-router-dom";
+import { useParams } from "react-router";
+import {
+  Card,
+  Flex,
+  FlexItem,
+  CardBody,
+  Title,
+  Text,
+  Popover,
+  Button,
+  TextVariants,
+  CardTitle
+} from "@patternfly/react-core";
+import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
+import { StyleSheet, css } from "aphrodite";
+export interface IConnectionGatewayProps {
```

Comment:
Are you using this interface in any other component, if not, please don't export it.